Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM busybox
+
+# The commit sha we wish to ship to prod
+ENV SHA master
+
+COPY . /app/phabricator/src/extensions/
```

Comment:
might as well use a version.json and keep it consistent, rather than rolling something new.